src/map/battle.cpp

```diff
@@ -411,7 +411,14 @@ int battle_delay_damage(t_tick tick, int amotion, struct block_list *src, struct
 	dat->additional_effects = additional_effects;
 	dat->src_type = src->type;
 	dat->isspdamage = isspdamage;
-	if (src->type != BL_PC && amotion > 1000)
+	if (skill_id == 0 && src->type == BL_MOB && battle_config.synchronize_damage) {
+		mob_data* md = BL_CAST(BL_MOB, src);
+		// The client refuses to display animations slower than 1x speed
+		// So we need to shorten AttackMotion to be in-sync with the client in this case
+		if (amotion > md->status.clientamotion)
```

Comment:
```suggestion
		if (md != nullptr && amotion > md->status.clientamotion)
```

Reply:
I was wondering if md can be nullptr at all because I check for bl->type beforehand.